mifospay/src/main/java/org/mifos/mobilewallet/mifospay/utils/Constants.java

```diff
@@ -181,6 +181,8 @@
 
     public static final String ERROR_FIELDS_CANNOT_BE_EMPTY = "Fields cannot be empty";
     public static final String ERROR_VALIDATING_PASSWORD = "Passwords are not the same";
```

Comment:
Change this OLD_PASSWORD_NEW_PASSWORD to something more meaningful, which reflects that it is an error message.